host/4/dotnet-build.yml

```diff
@@ -167,6 +167,20 @@ steps:
     displayName: dotnet-isolated-appservice (.NET 7 Preview)
     continueOnError: false
 
+  - bash: |
+      set -e
+      IMAGE_NAME=azurefunctions.azurecr.io/azure-functions/4/dotnet-isolated:$(Build.SourceBranchName)-dotnet-isolated7.0-mariner
+      IMAGE_ARRAY="$(image_list),$IMAGE_NAME"
+      echo "##vso[task.setvariable variable=image_list;]$IMAGE_ARRAY"
+
+      docker build -t $IMAGE_NAME \
+                  -f host/mariner/dotnet/dotnet7-isolated-appservice.Dockerfile \
+                  host/mariner/dotnet/
+      npm run test $IMAGE_NAME --prefix  test/
```

Comment:
Ain't no way this works. Just guessing

Reply:
Thought I pushed the change for that, not sure why it's not showing up